### PR TITLE
feat(color-picker, color-select-panel): [color-picker,color-select-panel] format

### PR DESCRIPTION
**Pull request by GaoNeng-wWw**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

- fix(color-picker): drop .black click
  - 不知道为什么如果加上 page.locator('.black')会一直死循环
- fix(color-picker): onCancel will not reset color
  - 由于 #2529 onCancel 不需要再reset color
- refactor: remove color utils
  - 仅仅是删除了遗留文件
- 该PR会在 #2529 合并后将WIP切换至ready

## Summary by CodeRabbit

- **New Features**
	- Introduced a new color selection component that allows users to choose between different color formats (hex, rgb, hsl, hsv) with a user-friendly interface.
	- Added functionality to toggle the visibility of predefined colors and history within the color picker and color select panel.
	- Updated demo entries to showcase the new color format functionality and visibility toggles for history and predefined colors.
- **Bug Fixes**
	- Updated test cases for the color picker and color select panel to ensure correct formatting of color values in different representations.
	- Enhanced test scripts to improve accessibility and interaction with UI elements.
- **Refactor**
	- Simplified the `onCancel` functionality within the event handling logic for better clarity and performance.
	- Enhanced the reactivity of state management related to predefined colors and history.
- **Chores**
	- Removed outdated utility functions and classes related to color manipulation to streamline the codebase.

### CODE REVIEW

**Comment by zzcr** [comment]
你好，这个mr主要解决那个issue的问题呢？可否贴一个效果图呢？

**Comment by zzcr** [comment]
是否是解决这个issue问题单：https://github.com/opentiny/tiny-vue/issues/2514

**Comment by GaoNeng-wWw** [comment]
> 是否是解决这个issue问题单：#2514

并不是，因为仅仅是暴露了一个 `format` 的 props 并且添加了示例

image

但是也可以加进去，不过我觉得这个PR似乎有些太大了

**Comment by GaoNeng-wWw** [comment]
要不还是加一下吧，先切换到 draft 了

File: examples/sites/demos/apis/color-picker.js [hunk 1]
```diff
@@ -60,6 +82,18 @@ export default {
           },
           mode: ['pc'],
           pcDemo: 'default-visible'
+        },
+        {
+          name: 'format',
+          type: "('hex'|'rgb'|'hsl'|'hsv')[]",
```
Comment (zzcr) [inline examples/sites/demos/apis/color-picker.js:94]: 新增的api需要标注下是哪个版本添加的哈，具体可以参考：https://github.com/opentiny/tiny-vue/wiki/%E5%AE%98%E7%BD%91%E7%BB%84%E4%BB%B6%E4%BB%A5%E5%8F%8AAPI%E7%89%88%E6%9C%AC%E7%8A%B6%E6%80%81%E7%BB%B4%E6%8A%A4%E4%BF%A1%E6%81%AF%E7%AE%A1%E7%90%86%E6%96%B9%E6%A1%88

Comment (GaoNeng-wWw) [reply examples/sites/demos/apis/color-picker.js:94]: 已解决

**Review by zzcr — APPROVED** [review]

**Pull request merged**